src/modules/commander/PreflightCheck.cpp

```diff
@@ -427,11 +413,11 @@ static bool airspeedCheck(orb_advert_t *mavlink_log_pub, vehicle_status_s &statu
 	}
 
 	/**
-	 * Check if differential pressure is off by more than 15Pa which equals ~5m/s when measuring no airspeed.
+	 * Check if airspeed is higher than 4m/s (accepted max) while the vehicle is landed / not flying
 	 * Negative and positive offsets are considered. Do not check anymore while arming because pitot cover
 	 * might have been removed.
 	 */
-	if (fabsf(differential_pressure.differential_pressure_filtered_pa) > 15.0f && !prearm) {
+	if (fabsf(airspeed.indicated_airspeed_m_s) > 4.0f && !prearm) {
```

Comment:
For what it is worth: 
We have been flying a lightly modified version of v1.8 firmware since June 2018 that includes code changes almost identical to what you are proposing in this PR.  One key difference we we also removed the "&& !prearm" requirement.  This effectively adds another safety check to arming that prevents cases where plugged pitot and bad calibrations were able to sneak through the preflight checks.   We experienced quite a few of these cases that where able to sneak through, eventually resulting in crashes.  
There are a couple major downsides to removing "&& !prearm"
1) some users might use pitot covers that are removed before arming, e.g. on a launcher or runway.
2) this effectively blocks disarming and re-arming in flight, which is a handy capability to have for a fixed wing.
I raise these points because it would be neat to be able to both improve arming safety while at the same time not blocking launcher/runway users and re-arming in flight.  Food for thought, either for this PR or potential future PRs.